packages/cavif-rs/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE="https://lib.rs/cavif"
+TERMUX_PKG_DESCRIPTION="Encoder/converter for AVIF images. Based on rav1e and avif-serialize."
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.3.0
+TERMUX_PKG_SRCURL=https://github.com/kornelski/cavif-rs/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=1ee6198b2dc0843cddaeeaece964d2615ca84997265b72ff69ca3ef9e18ddf23
+TERMUX_PKG_BUILD_DEPENDS="nasm"
```

Comment:
`nasm` is x86 assembler program. Therefore:
* If package really uses `nasm` during build time, it can't be compiled for ARM and AArch64.
* For i686 and x86_64 you don't need to specify it here as well. Termux build environment is Ubuntu and can't run packages compiled for Android. Add this utility to `scripts/setup-ubuntu.sh` instead.

Reply:
Seems like related to this:
```
error: failed to run custom build command for `rav1e v0.5.0-alpha`

Caused by:
  process didn't exit successfully: `/home/builder/.termux-build/cavif-rs/src/target/release/build/rav1e-4273389ee7ea02ba/build-script-build` (exit code: 101)
  --- stdout
  cargo:rustc-cfg=nasm_x86_64

  --- stderr
  thread 'main' panicked at 'This version of NASM is too old: No such file or directory (os error 2)', /home/builder/.cargo/registry/src/github.com-1ecc6299db9ec823/rav1e-0.5.0-alpha/build.rs:255:7
```

As I wrote - add nasm to `scripts/setup-ubuntu.sh` since it has to be executed on host and remove it from build depends.